src/zcl_aff_writer.clas.abap

```diff
@@ -430,11 +430,11 @@ CLASS zcl_aff_writer IMPLEMENTATION.
 
 
   METHOD call_reader_and_decode.
-    DATA(reader) = NEW cl_oo_abap_doc_reader( ).
+    DATA(ref) = cl_oo_factory=>create_instance( )->create_clif_source( name_of_source ).
+    ref->get_source( IMPORTING source = DATA(lt_source) ).
+    DATA(reader) = zcl_aff_abap_doc_reader=>create_instance( lt_source ).
```

Comment:
Can we remove the prefix of the local variable? We usually don't use any prefixes in this repo.

```suggestion
    ref->get_source( IMPORTING source = DATA(source) ).
    DATA(reader) = zcl_aff_abap_doc_reader=>create_instance( source ).
```